src/app/features/ui/filter-search-button/filter-search-button.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container" (appClickOutside)="hideDropdown()" appClickOutside>
```

Comment:
Nie ogarniam tej linijki, tzn. już widzę, że korzystasz tutaj z dyrektywy (swoja droga bardzo fajna opcja) ale nie rozumiem czemu jest appClickOutside bez nawiasów, to ma jakieś znaczeniu przy używaniu takich dyrektyw?

Reply:
appClickOutside bez nawiasów jest nie potrzebny, dzięki za uwagę. Już go usunąłem.